openedx/edx-platform-tutor.md

```diff
@@ -0,0 +1,156 @@
+---
+parent: OpenedX
+---
+# Platform Development - Tutor
```

Comment:
Similar to the devstack file, we can change this to `MITx & Open edX Integration with Tutor`.